shell/src/main/java/alluxio/cli/fs/command/DistributedMvCommand.java

```diff
@@ -50,31 +44,22 @@ public String getCommandName() {
 
   @Override
   public void validateArgs(CommandLine cl) throws InvalidArgumentException {
-    CommandUtils.checkNumOfArgsEquals(this, cl, 2);
+    mCpCommand.validateArgs(cl);
   }
 
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
+    mCpCommand.run(cl);
```

Comment:
If this can throw an exception, should we try to catch it, to let the user know that the copy phase did not complete, so it might be in an inconsistent state?

If it is in an inconsistent state, what happens if we run distributedMv again?

Reply:
distributedMv fails because overwrite is broken. But at the very least, you can delete the destination directory and then rerun distributedMv